mwdb/resources/group.py

```diff
@@ -248,6 +251,13 @@ def put(self, name):
         if obj["workspace"] is not None:
             group.workspace = obj["workspace"]
 
+        if (
+            user is not None
+            and user.login == group.name
+            and Capabilities.manage_users not in group.capabilities
+        ):
+            raise Forbidden(f"Can't remove '{Capabilities.manage_users }', yourself")
+
```

Comment:
1) You don't have to check if `user` is `None` - decorator `requires_authorization` does it for you

2) I think it would be much better to move this condition before `group.capabilities` is changed -> check values in `obj["capabilities"]` before line 246

Reply:
Now it look fine